storm-core/src/jvm/org/apache/storm/logging/ThriftAccessLogger.java

```diff
@@ -24,4 +26,10 @@
   public void log(String logMessage) {
     LOG.info(logMessage);
   }
+
+  public static void logAccess(Integer requestId, InetAddress remoteAddress, Principal principal, String operation) {
+    new ThriftAccessLogger().log(
+      String.format("Request ID: {} access from: {} principal: {} operation: {}",
+                    requestId, remoteAddress, principal, operation));
+  } 
```

Comment:
I thought it would be something like
```
LOG.info("Request ID: {} access from: {} principal: {} operation: {}",
                 requestId, remoteAddress, principal, operation);
```

and ThriftAccessLogger.log would just go away